mimic/model/identity.py

```diff
@@ -0,0 +1,22 @@
+"""
+Models relating to identity.
+"""
+from attr import attributes, attr, validators
```

Comment:
We haven't had a great process for vetting new requirements, so I wouldn't reject this patch based on the addition of `attrs` alone, but it does seem like if you'd like to land this, we should at least have a plan (filed issues) to migrate to `attrs` rather than depend on both it and `characteristic` long-term.